### PR TITLE
Add README and type documentation

### PR DESCRIPTION
**Pull request by dlh01**
## Summary

As titled.

## Notes for reviewers

None.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,31 @@
...
+Learn more about the objects included with Type Extensions in the [documentation for each type](#documentation).
+
+## Documentation
+
+- [Feature](docs/feature.md)
+- [Post IDs](docs/post-ids.md)
+- [Post Queries](docs/post-queries.md)
+- [Post Query](docs/post-query.md)
+- [Serialized Blocks](docs/blocks.md)
+- [Single Block](docs/blocks.md)
```
Comment (mogmarsh) [inline README.md]: I only see `docs/single-block.md`

Comment (dlh01) [reply README.md]: Fixed! Thanks.

File: docs/serialized-blocks.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+- [Blocks](https://github.com/alleyinteractive/wp-type-extensions/blob/main/src/alley/wp/blocks/class-blocks.php): Bundle many blocks.
+- [Each_Replaced_Blocks](https://github.com/alleyinteractive/wp-type-extensions/blob/main/src/alley/wp/blocks/class-each-replaced-blocks.php): Replace each matched block with other block content.
+- [Matched_Blocks](https://github.com/alleyinteractive/wp-type-extensions/blob/main/src/alley/wp/blocks/class-matched-blocks.php): Blocks matched with `match_blocks()`.
+- [Block_Content](https://github.com/alleyinteractive/wp-type-extensions/blob/main/src/alley/wp/blocks/class-block-content.php): Blocks in the given content.
+- [Lazy_Blocks](https://github.com/alleyinteractive/wp-type-extensions/blob/main/src/alley/wp/blocks/class-lazy-blocks.php): Instantiate blocks only when called upon.
+
+All `Single_Block` implementations also implement `Serialized_Blocks`.
+
+- [Inner_Blocks_Prepended](https://github.com/alleyinteractive/wp-type-extensions/blob/main/src/alley/wp/blocks/class-inner-blocks-prepended.php):
```
Comment (mogmarsh) [inline docs/serialized-blocks.md]: why is this item separate from the rest of the list?

**Review by mogmarsh — APPROVED** [review]
👍

File: docs/serialized-blocks.md (hunk 2 above)
Comment (dlh01) [reply docs/serialized-blocks.md]: Copy-paste flub. It's included in `single-block.md`.

**Pull request merged**